mlir/lib/ExecutionEngine/ROCm/BackendUtils.cpp

```diff
@@ -46,15 +47,37 @@ using namespace mlir;
 
 static constexpr const char kRunnerProgram[] = "mlir-rocm-runner";
 static constexpr const char kRocmAgentEnumerator[] = "rocm_agent_enumerator";
+static constexpr const char targetTriple[] = "amdgcn-amd-amdhsa";
```

Comment:
`kTargetTriple`

Reply:
Done in 4981d203a6c728f3f0698b2eb2bd1b0bec86a190